content/developer/reference/user_interface.rst

```diff
@@ -7,5 +7,7 @@ User interface
 .. toctree::
     :titlesonly:
 
+    user_interface/view_record
```

Comment:
What is the purpose of this page?

IMHO, it should be merged into `views.rst`, which would be easily restructured if each view type is moved as a separate sub-page.

Reply:
View and record views are two separate things.
It is not wise to mix the semantics of the views with the record that contains them. That was the error before. Record options are not equal to view options. Putting them together is just confusing.

A record is any object and in this case it is a view. We could make a csv that would not change the view. We can go to the client database we see views with the arch, we don't see <record>....

Reply:
Okay, makes sense. It wasn't clear to me when scanning through the content that the distinction what about that, but again I didn't read everything yet.